packages/core/src/blockchain/Blockchain.utils.ts

```diff
@@ -7,8 +7,14 @@
 import { SubmittableResult } from '@polkadot/api'
 import { SubmittableExtrinsic } from '@polkadot/api/promise/types'
 import {
+  ErrorCode,
+  ERROR_TRANSACTION_DUPLICATE,
+  ERROR_TRANSACTION_OUTDATED,
+  ERROR_TRANSACTION_PRIORITY,
   ERROR_TRANSACTION_RECOVERABLE,
   ERROR_TRANSACTION_USURPED,
+  isSDKError,
+  SDKError,
 } from '../errorhandling/SDKErrors'
```

Comment:
can you import everything as SDKErrors ? It's how I currently export the SDKErrors from the utils package. If we don't want to expose every single error type directly from the utils package, that's how this would have to be handled.

Reply:
Yes good suggestion!